fvm/src/call_manager/default.rs

```diff
@@ -611,10 +611,6 @@ where
                     value: ret.cloned(),
                 }),
                 Err(abort) => {
-                    if let Some(err) = last_error {
-                        cm.backtrace.begin(err);
-                    }
-
                     let (code, message, res) = match abort {
                         Abort::Exit(code, message, NO_DATA_BLOCK_ID) => (
```

Comment:
This (and below) may actually indicate "success". This has _changed_ since FVM v2 because we now support non-local exits.